.github/workflows/ci.yml

```diff
@@ -2,23 +2,29 @@ name: CI
 
 on:
   push:
-    branches: main
-    tags: v*
+    branches: [main]
+    tags: ["v*"]
   pull_request:
-    branches: "*"
+    branches: ["*"]
 
 jobs:
-  push:
-    name: push
+  build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 14.x
+          - 16.x
+          - 18.x
```

Comment:
Might as well test `20.x` so we catch the next problem :)

Reply:
<img width="865" alt="image" src="https://user-images.githubusercontent.com/14237/204613019-7d9432bf-6037-4fbf-a2d5-ddc57a4351ea.png">